UefiTestingPkg/AuditTests/PagingAudit/Windows/MemoryRangeObjects.py

```diff
@@ -135,10 +147,52 @@ def GetSystemMemoryType(self):
         if self.SystemMemoryType is None:
             return "None"
         try:
+            # Indices 5, 6, and 7 corrolate with a formatted string in the SystemMemoryTypes list.

```

Comment:
Small spelling error.  Should be correlate.

Reply:
Fixed